src/resources/i18n-target.html

```diff
@@ -0,0 +1,52 @@
+<template>
+
+  <button id="locale-changer-de" click.delegate="changeLocale('de')">DE</button>
+
+  <span id="i18n-simple" t="simple.text;[title]simple.attr"></span>

```

Comment:
simple is actually performing two tests on multiple features:

* simple interpolation
* attribute overrides
* multiple targets with semicolon

Reply:
Done